packages/@markuplint/ml-core/src/ml-rule/ml-rule.ts

```diff
@@ -16,6 +16,18 @@ export class MLRule<T extends RuleConfigValue, O = null> {
 	#v: MLRuleOptions<T, O>['verify'];
 	#f: MLRuleOptions<T, O>['fix'];
 
+	/**
+	 * The following getters are unused internally,
+	 * only for extending from 3rd party library
+	 */
+	get v() {
+		return this.#v;
+	}
+
+	get f() {
```

Comment:
@JounQin 

Please do the same here.

Reply:
OK, I was thinks `The following getters are` indicates that already. 😅

Reply:
@JounQin 

I know but some IDE cannot refer if do that so.